### ISSUE TITLE
Edit help text of question archive_v1.1.0-beta.1/legal-and-ethics/sensitive-data-personal_data_yesno/yesno

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#64]
**Describe the request**  
Add new related question

**Propose what exactly should be changed**  
Protective measures should be listed

**Proposed setting or content**   

*question text en*  
"If yes: please describe how they are protected in accordance with the mentioned data protection laws."
*help text en*
"Please describe the measures taken to protect the data."

*question text ger*
"Wenn ja: bitte beschreiben Sie, wie diese im Sinne der genannten Datenschutz-Gesetze geschützt werden."
*help text ger*
"Bitte führen Sie die Maßnahmen zum Schutz der Daten auf."

**Additional context**
new attribute needed.

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] help text (en and ger) for legal-and-ethics/sensitive-data-personal_data_yesno/yesno

**Comment by jmfrenzel** [comment]
There exists an attribute which can be reused
- [x] attribute project/dataset/data_security/security_measures

**Comment by smohex** [comment]
UDE/TUDO: we both agree with the suggestion

**Comment by smohex** [comment]
The attribute mentioned (project/dataset/data_security/security_measures) is connected with the question "Welche Maßnahmen zur Gewährleistung der Datensicherheit werden getroffen (z.B. Schutz vor unbefugtem Zugriff, Datenwiederherstellung, Übertragung sensibler Daten)?"
In issue #63 the question should be changed in „Falls die Daten nicht offen bereitgestellt werden können, durch welche Maßnahmen wie wird der eingeschränkte Zugriff sichergestellt?“ Attribut (project/dataset/data_security/access_permissions)
The questions seem very similar in my opinion. Should I change both like this?
Both attributes are not yet used in the archive catalog. Should both then be inserted there as well?

**Comment by jmfrenzel** [comment]
> The attribute mentioned (project/dataset/data_security/security_measures) is connected with the question "Welche Maßnahmen zur Gewährleistung der Datensicherheit werden getroffen (z.B. Schutz vor unbefugtem Zugriff, Datenwiederherstellung, Übertragung sensibler Daten)?"


> In issue #63 the question should be changed in „Falls die Daten nicht offen bereitgestellt werden können, durch welche Maßnahmen wie wird der eingeschränkte Zugriff sichergestellt?“ Attribut (project/dataset/data_security/access_permissions)
> The questions seem very similar in my opinion. Should I change both like this?

There is a technical problem. Both of the mentioned attributes are under `project/dataset/...` which is a collection while `legal-and-ethics/...` isn't a collection and can not be combined in a question set.   

> Both attributes are not yet used in the archive catalog. Should both then be inserted there as well?
In my opinion both should be added. However,  they would not appear in the same question set (see my comment above). 


As a possible solution I would suggest that instead of adding a new question (see issue description) we add the suggested content (text and help) as new help text to the question ` https://rdmorganiser.github.io/terms/questions/ua_ruhr/legal-and-ethics/sensitive-data-personal_data_yesno/yesno`  
**Proposed setting or content**   
*help text en*  
"If yes: Please also check whether you have completed section "Data usage / Data storage and security" with the appropriate measures to protect the data in accordance with the applicable data protection laws."

*help text ger*
"Wenn ja: Bitte prüfen Sie auch, ob Sie den Abschnitt "Datennutzung / Datenspeicherung und -sicherheit" mit den entsprechenden Maßnahmen zum Schutz der Daten gemäß den geltenden Datenschutzgesetzen ausgefüllt haben."

**Comment by smohex** [comment]
changed the help text as suggested

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/79

**Comment by ClaraRosalie** [comment]
implemented in upstream via rdmorganiser#83
